.github/workflows/contentful-backup.yml

```diff
@@ -0,0 +1,115 @@
+name: Contentful Backup
+on:
+  pull_request:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Which Contentful environment to backup
+        required: true
+        type: choice
+        options: [ 'Dev', 'Tst', 'Staging', 'Production' ]
+        default: 'Staging'
+
+jobs:
+  backup_content:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment || 'Dev' }}
+    name: Backup content for ${{ inputs.environment || 'Dev' }}
```

Comment:
This will need to default to Prod or Staging to auto backup the contentful content
So not sure the best option, We could put the contentful secrets into github secrets so it doesn't have to ask permission, or make this a workflow dispatch only, but I'd prefer it to be running on schedule